README.md

```diff
@@ -24,10 +24,16 @@ You can also *write* a custom script.
 ### Building and testing
 
 * Build the image with `rake docker:build`.
+* You'll need some environment variables in a file named `env.dev`. You'll need to
+  generate your own secrets:
```

Comment:
Are there instructions for generating these secrets?

Reply:
unfortunately, the slack secrets involve creating your own app, which seems like way too much work for just fixing a typo. What I'd like to do is figure out how to do a stub connector so it doesn't need a real slack connection to just test bot interactions.